src/Compilers/CSharp/Test/Symbol/SymbolDisplay/SymbolDisplayTests.cs

```diff
@@ -7806,5 +7806,51 @@ class A {
                 format,
                 "delegate*<in Int32, ref readonly String>");
         }
+
+        [Fact]
+        public void TestSynthesizedAnonymousDelegateType1()
+        {
+            var source = @"
+class C
+{
+    void M()
+    {
+        var v = (ref int i) => i.ToString();
```

Comment:
Could you include a similar test for a "simple signature" (no `ref`)?

Reply:
Sure!